include/ginkgo/core/base/executor.hpp

```diff
@@ -104,12 +104,42 @@ constexpr allocation_mode default_hip_alloc_mode =
 inline namespace cl {
 namespace sycl {
 
+
 class queue;
 
+
 }  // namespace sycl
 }  // namespace cl
 
 
+/**
+ * The enum class for the dpcpp queue property.
+ */
```

Comment:
I would like to have a comment that it is possible (and legal) to combine multiple properties with a binary or (`|`).